### PR TITLE
Write geometry column for write demographic data

### PR DESCRIPTION
**Pull request by justinebreuch**
## Description

Addresses: [Shortcut story](https://app.shortcut.com/blueconduit/story/4764/census-block-lsl-data-is-stored-in-the-db)

Writes a single polygon to all rows being written by the demographic lambda.

### New

Geometry column write

### Changed

Removed bulkInsert because it does not elegantly handle custom types 

## Testing and Reviewing

Test event inside with AWS console

### CODE REVIEW

**Comment by justinebreuch** [comment]
Do we have a common formatter? I'm updating things I don't mean to by my intelliJ formatting

File: cdk/src/open-data-platform/lambda/data-import-stack.ts [hunk 1]
```diff
@@ -19,27 +21,27 @@ export class DataImportStack extends Construct {
...
-          CREDENTIALS_SECRET: credentialsSecret.secretArn,
-          DATABASE_NAME: db
-        },
-        timeout: Duration.minutes(5),
-        bundling: {
-          externalModules: ['aws-sdk'],
-          nodeModules: ['csv-parser', '@databases/pg', '@databases/pg-bulk'],
-        },
-      });
+        new lambda.NodejsFunction(this, 'write-demographic-data-handler', {
```
Comment (kailamjeter) [inline cdk/src/open-data-platform/lambda/data-import-stack.ts:24]: Are the styling changes intentional? I had to add an intellij extension (Prettier) to get intellij to reformat based on the .prettierrc.

File: cdk/src/open-data-platform/lambda/write-demographic-data.handler.ts [hunk 2]
```diff
@@ -75,36 +75,51 @@ async function connectToDb(secretArn: string): Promise<ConnectionPool> {
...
-        database: db,
-        tableName: `demographics`,
-        columnTypes: {
-          census_geo_id: sql`VARCHAR`,
-          total_population: sql`REAL`,
-          black_percentage: sql`REAL`,
-          white_percentage: sql`REAL`,
-        },
-      };
+  // TODO(breuch): Replace with geom from new file.
```
Comment (kailamjeter) [inline cdk/src/open-data-platform/lambda/write-demographic-data.handler.ts:78]: Can you mention in this comment what this raw value is?

**Review by kailamjeter — APPROVED** [review]

File: cdk/src/open-data-platform/lambda/data-import-stack.ts (hunk 1 above)
Comment (justinebreuch) [reply cdk/src/open-data-platform/lambda/data-import-stack.ts]: Just added it and ran on files. Thank you!

File: cdk/src/open-data-platform/lambda/write-demographic-data.handler.ts (hunk 2 above)
Comment (justinebreuch) [reply cdk/src/open-data-platform/lambda/write-demographic-data.handler.ts]: Done!

File: cdk/src/open-data-platform/lambda/data-import-stack.ts [hunk 3]
```diff
@@ -1,45 +1,51 @@
...
         vpc: vpc,
-        vpcSubnets: {subnetType: ec2.SubnetType.PRIVATE_WITH_NAT},
+        vpcSubnets: { subnetType: ec2.SubnetType.PRIVATE_WITH_NAT },
         environment: {
           CREDENTIALS_SECRET: credentialsSecret.secretArn,
-          DATABASE_NAME: db
+          DATABASE_NAME: db,
         },
-        timeout: Duration.minutes(5),
+        timeout: Duration.minutes(10),
```
Comment (beekley) [inline cdk/src/open-data-platform/lambda/data-import-stack.ts:35]: Any reason for the number of minutes here?

File: cdk/src/open-data-platform/lambda/data-import-stack.ts [hunk 4]
```diff
@@ -55,8 +61,11 @@ export class DataImportStack extends Construct {
 
     if (writeDemographicDataFunction.role?.roleArn != undefined) {
       let role = iam.Role.fromRoleArn(
-        scope, id + '-role', writeDemographicDataFunction.role.roleArn);
+        scope,
+        id + '-role',
```
Comment (beekley) [inline cdk/src/open-data-platform/lambda/data-import-stack.ts:65]: nit: I find string templates like `` `${id}-role` `` to be more readable than using the `+` operator, which might do some weird type casting.

**Review by beekley — APPROVED** [review]

File: cdk/src/open-data-platform/lambda/data-import-stack.ts (hunk 3 above)
Comment (justinebreuch) [reply cdk/src/open-data-platform/lambda/data-import-stack.ts:35]: It was timing out

**Pull request merged**